SRC/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo List</title>
+  </head>
+  <body>
+    <div class="main-div">
+      <input type="text" class="ibput-box" />
+      <h2>today todos</h2>
+      <ul class="the-list"></ul>
+      <button class="clear-btn">Clear all completed</button>
+    </div>
```

Comment:
- [ ] Your project should be styled to look exactly like the given design. Kindly ensure you update the styling.

| Original | Yours
| ------- | ---------- |
| <img src='https://user-images.githubusercontent.com/33323077/125591073-122c8f39-105d-4cb4-85e4-0afea32f2186.png' width='250px' /> | <img src='https://user-images.githubusercontent.com/33323077/125591214-7e140437-7f32-4742-8ea8-539959e949c8.png' width='250px' /> |